### PR TITLE
Project Metrics: Add release-granularity metric

### PR DESCRIPTION
**Pull request by rcebulko**

### CODE REVIEW

File: project-metrics/metrics_service/metrics/release_granularity.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    elif avg_commits > 100:
+      return models.MetricScore.MODERATE
+    elif avg_commits > 10:
+      return models.MetricScore.GOOD
+    else:
+      return models.MetricScore.EXCELLENT
+
+  def _compute_value(self) -> float:
+    """Computes the average number of commits between releases over the last 90
+
```
Comment (danielrozenberg) [inline project-metrics/metrics_service/metrics/release_granularity.py]: nit: unnecessary extra space

File: project-metrics/metrics_service/metrics/release_granularity.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    Returns:
+      The percentage of passing builds.
+    """
+    logging.info('Counting commits per release')
+    session = db.Session()
+    releases = session.query(models.Release).filter(
+        models.Release.is_last_90_days()).all()
+    release_count = len(releases)
+
+    if release_count == 0:
```
Comment (danielrozenberg) [inline project-metrics/metrics_service/metrics/release_granularity.py]: You can just keep the `< 2` case, the error message is correct for both ;)

File: project-metrics/metrics_service/metrics/release_granularity.py (hunk 1 above)
Comment (rcebulko) [reply project-metrics/metrics_service/metrics/release_granularity.py]: Done

File: project-metrics/metrics_service/metrics/release_granularity.py (hunk 2 above)
Comment (rcebulko) [reply project-metrics/metrics_service/metrics/release_granularity.py]: Done.

File: project-metrics/metrics_service/cron.yaml [hunk 3]
```diff
@@ -16,6 +16,10 @@ cron:
   url: /_cron/recompute/PresubmitLatencyMetric
   schedule: every 24 hours
 
+- description: "Update release-granularity metric."
+  url: /_cron/recompute/ReleaseGranularityMetric
+  schedule: every 72 hours
```
[the remark this answers is not in the document]
Comment (rcebulko) [reply project-metrics/metrics_service/cron.yaml]: Done

**Pull request merged**